cli/lock.ml

```diff
@@ -548,12 +548,12 @@ let info =
          instructions there to add dune ports for the packages you need.";
     ]
   in
-  Term.info "lock" ~doc ~exits ~man
+  Cmd.info "lock" ~doc ~exits ~man
 
 let term =
   let open Term in
```

Comment:
Maybe while at it, avoid the `open Term` and use `Term.(const run …)`? With multiple modules named `Term` it becomes a bit less obvious what should be opened and what not. `pull.ml` does this that way and I think it is more readable and understandable.